hooks/build

```diff
@@ -0,0 +1,10 @@
+!/bin/bash
+
+# Docker Daemon Build Hook
+# $IMAGE_NAME var is injected into the build so the tag is correct.
+
+docker build \
+		--build-arg BUILD_DATE=`date -u +"%Y-%m-%dT%H:%M:%SZ"` \
```

Comment:
Backticks and date format should be refactored.

Reply:
DONE